bin/bridge-client/src/modes/operator/task_manager.rs

```diff
@@ -28,6 +28,7 @@ where
     pub(super) broadcaster: Arc<Bcast>,
     pub(super) bridge_duty_db_ops: Arc<BridgeDutyOps>,
     pub(super) bridge_duty_idx_db_ops: Arc<BridgeDutyIndexOps>,
+    pub(super) max_retry_count: u32,
```

Comment:
Could wrap this in a `Config` type so that we don't have to keep adding args to startup functions if we want to add more.

Reply:
It's just this one right now so introducing a struct is a bit of an overkill. But will definitely do that once this starts to become untenable.